### DISCUSSION TITLE
[New3D] Toggle object outlines scene-wide

### DISCUSSION THREAD

**Discussion by jhurliman** [foxglove/community#280] [Q&A]
The new 3d panel draws outlines around markers and some other renderables by default. There should be a setting under `Scene` to toggle these on or off.

![outlines](https://user-images.githubusercontent.com/195374/177862533-142b278a-096e-4ba6-b07f-f23301cefcae.png)

**Comment by amacneil** [comment]
![image](https://user-images.githubusercontent.com/637671/177916293-c1910c0b-3bcf-4ce8-a5ae-a003316784f3.png)
